internal/homing/homing.go

```diff
@@ -0,0 +1,117 @@
+package homing
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/mainflux/mainflux/logger"
```

Comment:
You can use this approach as suggested here https://github.com/mainflux/mainflux/pull/1748
```suggestion
	mflog "github.com/mainflux/mainflux/logger"
```